sites/partners/src/components/applications/PaperApplicationForm/sections/FormMultiselectQuestions.tsx

```diff
@@ -106,6 +122,10 @@ const FormMultiselectQuestions = ({
               stateKeys={stateKeys}
               data-testid={"app-question-extra-field"}
             />
+            <MultiselectQuestionsMap
```

Comment:
This is an issue that already exists, but when a user selects the "I don't want this preference" it doesn't remove the other checked preference options and hide the address fields. Once again, this doesn't have to be handled as part of this PR, but something to note.